test/common/src/main/kotlin/org/codice/compliance/verification/binding/RedirectBindingVerifier.kt

```diff
@@ -41,10 +45,50 @@ class RedirectVerifier(val response: IdpRedirectResponse) : BindingVerifier() {
         }
     }
 
+    /**
+     * Verifies the encoding of the samlResponse by decoding it according to the redirect binding rules in the binding spec
+     * 3.4.4.1 Deflate Encoding
+     */
+    private fun decodeAndVerify() {
```

Comment:
❗️ Add a `@Suppress("ComplexMethod")` to this method. The complex `when` for the exception parsing increases the complexity of this method.